lib/tasks/gitlab/bulk_add_permission.rake

```diff
@@ -44,5 +44,15 @@ namespace :gitlab do
         group.add_users(Array.wrap(user.id), GroupMember::DEVELOPER)
       end
     end
+
+    desc "GITLAB | Add a specific user to all groups (as a guest)"
```

Comment:
Use `GitLab` instead of `GITLAB` as per https://github.com/gitlabhq/gitlabhq/commit/b8f12d1917d2befe17cf8e8f47fb70a1bb6a99ff